spec/rspec/matchers/built_in/include_spec.rb

```diff
@@ -51,6 +51,14 @@
           expect("abc\ndef").to include("g", "h")
         }.to fail_matching("expected \"abc\\ndef\" to include \"g\" and \"h\"\nDiff")
       end
+
+      it "does not diff when lines match but are not an exact match" do
+        expect {
+          expect(" foo\nbar\nbazz").to include("foo", "bar", "gaz")
+        }.to raise_error(RSpec::Expectations::ExpectationNotMetError) { |error|
+          expect(error.message).not_to match(/Diff/)
+        }
+      end
```

Comment:
Good point!  I think maybe this will work:

``` ruby
RSpec::Matchers.define_negated_matcher :a_string_not_matching, :match

expect {
  expect(" foo\nbar\nbazz").to include("foo", "bar", "gaz")
}.to fail_with(a_string_not_matching(/Diff/i))
```

(The negated matcher def should probably go in `spec/support/matchers.rb`)